cmd/main.go

```diff
@@ -101,7 +100,6 @@ func executeRecoveryProcess(configuration config.Config, services Services, repo
 
 func initializeAPIRouter(services *Services) *apirouter.APIRouter {
 	apiRouter := apirouter.NewAPIRouter()
-	apiRouter.AddV1Router(metadata.Route, metadata.NewRouter(services.fees))
 	apiRouter.AddV1Router(healthcheck.Route, healthcheck.NewRouter())
 	apiRouter.AddV1Router(transaction.Route, transaction.NewRouter(services.transfers))
```

Comment:
nit: Wouldnt it be better if we rename the route from `transaction` to `transfer`